docs/build-the-docs.md

```diff
@@ -25,17 +39,20 @@ There, you can see:
 - The file it was found in
 - The line of that file where it is located
 
-## Viewing Changes
+## Viewing Changes
 
-Assuming that there are no errors, the next thing to do is to run the NPM web server so that you can view your changes, before committing and pushing the changes to the remote docs repository.
+Assuming that there are no errors, the next thing to do is to run the [NPM Serve tool](https://www.npmjs.com/package/serve) so that you can view your changes, before committing and pushing the changes to the remote docs repository.
+We suggest using it, as it's trivial to install and start.
+You could also use [PHP's built-in webserver](https://secure.php.net/manual/en/features.commandline.webserver.php) as well.
+By using either of these, you don't have to install and configure a complete webserver, such as Apache or NGINX.
 
-To do that, run the command:
+To start *Serve*, run the following command:
 
 ```
 serve public &
 ```
 
-This starts up the NPM web server, using the `public` directory, (re)generated by `antora site.yml`, as the document root, listening on `http://localhost:5000`.
+This starts Serve, using the `public` directory, (re)generated by `antora site.yml`, as the document root, listening on `http://localhost:5000`.
```

Comment:
``This starts Serve,`` --> ``This starts Serve and let it run in the background,``

Reply:
Done.